### ISSUE TITLE
Validation Accuracy

### ISSUE THREAD

**Issue by suchiek** [fizyr/keras-retinanet#419]
I am using train.py to do some transfer learning on my labelled data in a csv file. I froze the resnet backbone and used other default settings.

This is what I am seeing for trainable parameters:

Total params: 36,486,682
Trainable params: 12,925,530
Non-trainable params: 23,561,152

The model is still training and here is the output:

Epoch 9/50
 9999/10000 [============================>.] - ETA: 0s - loss: 0.7709 - regression_loss: 0.5119 - classification_loss: 0.2590                 
Epoch 00009: saving model to ./snapshots/resnet50_csv_09.h5
10000/10000 [==============================] - 5031s 503ms/step - loss: 0.7712 - regression_loss: 0.5121 - classification_loss: 0.2591
Epoch 10/50
 1157/10000 [==>...........................] - ETA: 1:14:00 - loss: 0.7830 - regression_loss: 0.5143 - classification_loss: 0.2687

Please let me know how I can see the validation accuracy. Do we need a validation csv file for this?

thanks!

**Comment by hgaiser** [comment]
Yes you need a validation CSV for validation accuracy, how else should it be computed? Check the `--val-annotations` argument.

**Comment by ShibaPrasad** [comment]
@hgaiser:  would you please tell me in details and command i need to use for  --val-annotations ?
